<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Chamfer_Dialog</class>
 <widget class="QDialog" name="Chamfer_Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>343</width>
    <height>260</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>343</width>
    <height>260</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>343</width>
    <height>260</height>
   </size>
  </property>
  <property name="font">
   <font>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Insert chamfer options</string>
  </property>
  <widget class="QPushButton" name="_2">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>210</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="_1">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>210</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>321</width>
     <height>91</height>
    </rect>
   </property>
   <property name="title">
    <string>Define chamfer by</string>
   </property>
   <widget class="QRadioButton" name="_100">
    <property name="geometry">
     <rect>
      <x>-30</x>
      <y>26</y>
      <width>231</width>
      <height>21</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::RightToLeft</enum>
    </property>
    <property name="text">
     <string>Length of chamfer</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="_101">
    <property name="geometry">
     <rect>
      <x>-30</x>
      <y>56</y>
      <width>231</width>
      <height>21</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::RightToLeft</enum>
    </property>
    <property name="text">
     <string>Distance from intersection</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>112</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Chamfer length</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>142</y>
     <width>203</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Chamfer distance along line 1</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>170</y>
     <width>201</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Chamfer distance along line 2</string>
   </property>
  </widget>
  <widget class="RCDoubleSpinBox" name="_102">
   <property name="geometry">
    <rect>
     <x>214</x>
     <y>110</y>
     <width>116</width>
     <height>22</height>
    </rect>
   </property>
   <property name="decimals">
    <number>6</number>
   </property>
   <property name="maximum">
    <double>100000000.000000000000000</double>
   </property>
  </widget>
  <widget class="RCDoubleSpinBox" name="_103">
   <property name="geometry">
    <rect>
     <x>214</x>
     <y>140</y>
     <width>116</width>
     <height>22</height>
    </rect>
   </property>
   <property name="decimals">
    <number>6</number>
   </property>
   <property name="maximum">
    <double>100000000.000000000000000</double>
   </property>
  </widget>
  <widget class="RCDoubleSpinBox" name="_104">
   <property name="geometry">
    <rect>
     <x>214</x>
     <y>170</y>
     <width>116</width>
     <height>22</height>
    </rect>
   </property>
   <property name="decimals">
    <number>6</number>
   </property>
   <property name="maximum">
    <double>100000000.000000000000000</double>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>RCDoubleSpinBox</class>
   <extends>QDoubleSpinBox</extends>
   <header>../RealCADQt/source/rcdoublespinbox.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>_102</tabstop>
  <tabstop>_103</tabstop>
  <tabstop>_104</tabstop>
  <tabstop>_1</tabstop>
  <tabstop>_2</tabstop>
  <tabstop>_101</tabstop>
  <tabstop>_100</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
